_data/fellow_projects/FAIR4UFOs.yml

```
@@ -0,0 +1,17 @@
+open: yes
+postdate: 2022-01-24
+title: Exploring the FAIR principles for preservation of UFO models
+description: >
+The FAIR (Findable, Accessible, Interoperable,  Reusable)  principles represent a set of standards
+that allow transparent and tractable progression in preservation of scientific data, tools, and software.
+One important set of tools used for Monte Carlo simulation of new physics models are the so called
+Universal Feynrules Outputs (UFO) models that are used in conjunction with event simulator like MadGraph.
+This project will be dedicated to developing a set of community standards in close association with the
+theorists, phenomenologists, and experimentalists to address FAIR preservation of such models.
+The project will encompass development of tools and CI-enabled repositories to allow a version controlled
+and well-documented preservation of these UFO models that can be then easily and reliably used by physicists.
```

These need to be indented. That's how YAML knows that the block is over and normal keeps start again.

```suggestion
  The FAIR (Findable, Accessible, Interoperable,  Reusable)  principles represent a set of standards
  that allow transparent and tractable progression in preservation of scientific data, tools, and software.
  One important set of tools used for Monte Carlo simulation of new physics models are the so called
  Universal Feynrules Outputs (UFO) models that are used in conjunction with event simulator like MadGraph.
  This project will be dedicated to developing a set of community standards in close association with the
  theorists, phenomenologists, and experimentalists to address FAIR preservation of such models.
  The project will encompass development of tools and CI-enabled repositories to allow a version controlled
  and well-documented preservation of these UFO models that can be then easily and reliably used by physicists.
```